common/djangoapps/util/milestones_helpers.py

```diff
@@ -376,9 +376,13 @@ def any_unfulfilled_milestones(course_id, user_id):
     """ Returns a boolean if user has any unfulfilled milestones """
     if not settings.FEATURES.get('MILESTONES_APP', False):
         return False
-    return bool(
-        get_course_milestones_fulfillment_paths(course_id, {"id": user_id})
-    )
+
+    from milestones import api as milestones_api
+    fulfillment_paths = milestones_api.get_course_milestones_fulfillment_paths(course_id, {'id': user_id})
+
```

Comment:
The other methods are using the API gateway, so I prefer keeping it like this. In the same hand, the `get_course_milestones_fulfillment_paths` function is rechecking the feature flag again, a reason that tells not call it.

Let's wait for edX's review and see. But we still need to know the status of the changes if we need had to merge this before edX do.